server/routes/loginStrategies/googleStrategy.js

```diff
@@ -33,6 +34,7 @@ passport.use(
             }
```

Comment:
Maybe log the fact that a user was successfully logged in using Google?  Something like `User (${user.id}) successfully logged in using Google`?